articles/client-platforms/react/05-linking-accounts.md

```diff
@@ -0,0 +1,372 @@
+            <Thumbnail src={profile.picture}>
+              <p>Welcome {profile.name}!</p>
+              <p>
+                <Button bsStyle="default" onClick={this.logout.bind(this)}>Logout</Button>
+              </p>
+            </Thumbnail>
+          </Col>
+          <Col md={4} className={styles.pane}>
+            <LinkedAccountsList profile={profile} auth={this.props.auth}></LinkedAccountsList>
+          </Col>
+        </Row>
+      </div>
+    )
+  }
+}
+
+export default Home;
+```
+
+Notice that `Home` is updated to render a left column with some profile info like name, avatar and a button to `logout`. The right columns is reserved to render the `LinkedAccountsList`. If you run the application you should see the new home page after a successful login, and the __Linked Accounts__ list shows only the main account. In the next topic we show how to add a button to link an account from an external provider.
+
+## 2. Linking Accounts
+
+To link accounts you basically need to hit the [link user account endpoint](https://auth0.com/docs/api/management/v2#!/Users/post_identities) in auth0 api. To complete the request, the primary auth0 JWT (the token provided when the user login), the user id (could be taken from the JWT or from profile api) and the JWT of the account you want to link (secondary account) should be provided.
+
+As you need to do a second login to get the secondary account JWT, we will use another instance of `Auth0Lock`, this time managed by a new helper class we're creating in `src/utils/LinkAccountService.js`.
+
+```javascript
+/* ===== ./src/utils/LinkAccountService.js ===== */
+import Auth0Lock from 'auth0-lock'
+
+export default class LinkAccountService {
+  constructor(auth) {
+    this.auth = auth
+    // the Auth0Lock instance to show signin window to link a provider
+    this.lock = new Auth0Lock(auth.clientId, auth.domain, {
+      auth: {params: {state: 'linking'}}, // state to identify in the callback
+      allowedConnections: ['facebook', 'google-oauth2'],
+      languageDictionary: { // allows to override dictionary entries
+        title: 'Link with:' // new window title
+      }
+    })
+    this.link = this.link.bind(this)
+  }
+
+  link(){
+    // Call the show method to display the authentication window.
+    this.lock.show()
+  }
+}
+```
+
+`AuthService` continues responsible to listen to `authenticated` events, but we need a way to determine if the callback was trigged by a regular sign in process or by a linking. That's why in the code above you see the new `Auth0Lock` instance is receiving specific options, in that case `auth: {params: {state: 'linking'}}`. For more information about supported options please
+[check the documentation](https://github.com/auth0/lock/tree/v10.0.0-rc.1#authentication-options). Next you see the updated `AuthService` class where `_doAuthentication` callback is checking for `linking` status and calling `LinkAccount` method instead.
```

Comment:
@mpaktiti I prefer to create another PR to update all the docs to the new version, because first we'll need to update in samples and test if everything is working as expected.